app/views/curation_concerns/base/_form_representative.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+  <fieldset id="representative-media">
+    <legend>
+      Representative Media
+    </legend>
+    <p>Select the file with media that represents this <%= f.object.human_readable_type %>.</p>
+    <%= f.select :representative_id, @form.select_files, {}, { class: 'form-control' } %>
```

Comment:
@jcoyne I copied the `f.object.select_files` from CurationConcerns. If I change the syntax here I won't be able to inherit from them as you suggested in your comment above.